packages/VendorLocker/ParentClassMethodTypeOverrideGuard.php

```diff
@@ -37,14 +37,20 @@ public function getParentClassMethod(ClassMethod $classMethod): ?MethodReflectio
 
         /** @var string $methodName */
         $methodName = $this->nodeNameResolver->getName($classMethod);
-        $parentClassReflections = array_merge($classReflection->getParents(), $classReflection->getInterfaces());
+        $parentClassReflection = $classReflection->getParentClass();
+        while  ($parentClassReflection !== null) {
```

Comment:
This should be instanceof check. I'll make a note to make Rector rule to cover these :)

Reply:
fixed, thanks